.codespellignore

```diff
@@ -0,0 +1,6 @@
+doubleclick
+thur
+neet
+bre
+wan
```

Comment:
I've submitted a PR which will remove some of these in due course:
https://github.com/codespell-project/codespell/pull/2337